samples/routing/flowcontrol/sample.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Fair load distribution
+summary: Implementing fair load distribution for heterogeneous scaled-out endpoints 
+component: Core
+tags:
+- Routing
+- Distribution
+- DistributionStrategy
+redirects:
+---
+
+## Running the project
+
+ 1. Start all the projects by hitting F5.
+ 1. The text `Press <enter> to send a message` should be displayed in the Client's console window. 
+ 1. Hit `<enter>` several times to send some messages.
+
+### Verifying that the sample works correctly
+
+ 1. Notice more messages are being send to Server.1 compared to Server.2
+ 1. Open MSMQ monitoring tool of your choice (e.g. QueueExplorer).
```

Comment:
Do we include references to specific tools in the doco? If so, should this be linked to the QueueExplorer home page?